waiting_room.go

```diff
@@ -39,6 +39,7 @@ type WaitingRoom struct {
 	CookieSuffix               string              `json:"cookie_suffix"`
 	AdditionalRoutes           []*WaitingRoomRoute `json:"additional_routes,omitempty"`
 	QueueingStatusCode         int                 `json:"queueing_status_code"`
+	EnabledOriginCommands      []string            `json:"enabled_origin_commands"`
```

Comment:
are you happy to always have this value sent? even when empty?

Reply:
It won't hurt, but we can omit it if it's empty.